HOWTO_android.md

```diff
@@ -0,0 +1,637 @@
+
+From the top-level samples folder you now need to enter the following sub-
+folder to build
+
+```bash
+cd build/android_gradle
+```
+
+### 8. Build the appropriate target
+
+Some common targets are `assembleDebug` and `assembleRelease`.
+We'll build for debug in this scenario:
+
+```bash
+gradle assembleDebug
+```
+
+
+## Capturing Vulkan Samples
+
+This example follows using the Khronos Vulkan Samples built above and assumes
+that you built the debug version.
+The instructions really only work on non-rooted phones if you manually build and
+install the application with support for the GfxReconstruct layer.
+The benefit of building the application in this way is that it will
+automatically incorporate the GfxReconstruct capture layer into the .APK file
+for the built application.
+
+### 1. Remove any previous version of the Vulkan Samples app
+
+We'll intentionally remove any previous instance of the application we're going
+to trace.
+In this case, the full name of the Vulkan samples app for Android is called
+"com.khronos.vulkan_samples":
+
+```bash
+adb uninstall com.khronos.vulkan_samples
+```
+
+### 2. Enable logcat in a separate terminal tab or window
+
+```bash
+adb logcat -s gfxrecon,vulkan
+```
+
+### 3. Install the debug version with storage access
+
+Install the debug version of the samples we just built with the
+`-g` option to request storage access:
+
+```bash
+adb install -g ./build/android_gradle/app/build/outputs/apk/debug/vulkan_samples-debug.apk
+```
+
+This should guarantee that the application (and any associated layers) is
+allowed to write out anything generated.
+
+### 4. Enable The GfxReconstruct layer
+
+Enable the GfxReconstruct layer just for the application using the global
+settings Android provides:
+
+```bash
+adb shell settings put global enable_gpu_debug_layers 1
+adb shell settings put global gpu_debug_app com.khronos.vulkan_samples
+adb shell settings put global gpu_debug_layers "VK_LAYER_LUNARG_gfxreconstruct"
+```
+
+### 5. Set the capture options
+
+Now, set the command to capture frames 500-700 so we can see what's happening
+and record the file out to something on the /sdcard folder:
+
+```bash
+adb shell "setprop debug.gfxrecon.capture_file  '/sdcard/samples_capture.gfxr'"
+adb shell "setprop debug.gfxrecon.capture_frames '500-700'"
+```
+
+More capture options can be found in the [USAGE_android.md](./USAGE_android.md)
+under the [Capture Options](./USAGE_android.md#capture-options) section.
+
+### 6. Run the application
+
+Keep the logcat output in a separate window/terminal because you can see
+useful information there.
+
+To run the application, I found it on the Android device in the application's
+list, then I clicked it to launch.
+Android may ask you to "Allow access to manage all files".
+When it asks you that, enable the option and click the back arrow to get to the
+application.
+
+Once open, you may see something similar to the following message in the
```

Comment:
You will definitely see this message, won't you?  How about "Once launched, you will see something similar to"